### PR TITLE
add license head for *.cs, *.ps1, *.sh files

### PR DESCRIPTION
**Pull request by Zongrenfeng**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the UWP implementation?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
Add license head for *.cs, *.ps1, *.sh files
excluded files:
 -- *.ps1, *.sh scripts come from cake;
 -- *.AssemblyInfo.cs
 -- *.Designer.cs
 -- files under root\tools, root\packages folders;

## Related PRs or issues
issue #1361 https://github.com/Microsoft/AppCenter-SDK-Apple/issues/1361/

### CODE REVIEW

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/MainPage.xaml.cs [hunk 1]
```diff
@@ -1,4 +1,7 @@
-﻿// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
```
Comment (ElektrojungeMS) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/MainPage.xaml.cs]: I'm missing the context for this additional line here but my guess it's fine. Leaving this comment to make sure other see it while they look at this PR.

Comment (achocron) [reply Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/MainPage.xaml.cs]: It was already in the code base, i think it's fine to leave as is.

File: SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.iOS.Bindings/Extensions.cs [hunk 2]
```diff
@@ -1,4 +1,7 @@
-﻿﻿using System;
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+?using System;
```
Comment (jwallra) [inline SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.iOS.Bindings/Extensions.cs]:
```suggestion
using System;
```

**Comment by guperrot** [comment]
@Zongrenfeng cake was our own logic, its a build tool but the scripts our own.

**Comment by guperrot** [comment]
Same for powershell logic, everything was written by us.

**Comment by guperrot** [comment]
I also included AssemblyInfo.cs that are regular cs files.

**Pull request merged**